src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/EntityMetadata.java

```diff
@@ -47,6 +48,18 @@ public EntityMetadata(ConnectionImpl connection) {
 				packets.addAll(itemDataCache.updateItem(version, entity.getId(), itemWatcher.getValue()));
 			});
 		}
+		//TODO: this can probably go in the remapper, but i couldnt get it to work there....
```

Comment:
this one mystified me a bit

Reply:
Oh no problem I will move it after merge.